storage/lmdb/src/lib.rs

```diff
@@ -180,21 +240,26 @@ impl backend::Backend for Lmdb {
         std::fs::create_dir_all(&self.path).map_err(error::process_io_error)?;
 
         // Set up LMDB environment
-        let mut env = lmdb::Environment::new();
-        env.set_max_dbs(desc.len() as u32);
-        env.set_flags(self.flags);
-        //env.set_map_size(todo!());
-        //env.set_max_readers(todo!());
-        let env = env.open(&self.path).or_else(error::process_with_err)?;
+        let environment = {
+            let mut env_builder = lmdb::Environment::new();
```

Comment:
I wouldn't make this mut then use it. Just put all the other commands after new().